requirements.txt

```diff
@@ -1,12 +1,16 @@
 Django==4.2.3
+wheel==0.41.1
+setuptools==68.1.0
+psycopg2>=2.8
+psycopg2-binary==2.9.7
+GDAL==3.7.1.1
 backports.zoneinfo;python_version<"3.9"
 asgiref==3.7.1
 django-s3sign==0.3.3
 pytz==2023.3
 httplib2==0.22.0
 feedparser==6.0.8
 Markdown==3.4.1 # pyup: < 3
-psycopg[binary]==3.1.9
```

Comment:
psycopg should be kept at version 3 - I don't think there is a reason to downgrade to version 2, unless postgis really has an issue with that. But, we're running version 3 just fine in production as is.